### PR TITLE
Add Cloudflare Workers icon

### LINKED ISSUE #8382: Cloudflare Workers
### Brand Name

Cloudflare Workers

### Website

https://workers.cloudflare.com

### Popularity Metric

There are 2 repositories that I would consider relevant:
https://github.com/cloudflare/workerd with 4.4k stars
https://github.com/cloudflare/workers-sdk with 1.1k stars

.. and maybe https://github.com/cloudflare/wrangler-legacy the old version of wrangler with 3.2k stars

### Official Resources for Icon and Color

https://workers.cloudflare.com has an inline svg.
The color is the same as Cloudflare and Cloudflare Pages, `#F38020`.

### Additional Comments

The set already contains Cloudflare Pages, and the Cloudflare logo.

Issue for Pages for reference: #7486

The Brand guidelines are shared between all of Cloudflare, so the links are identical.

### Contributing

Yes

### PR DESCRIPTION
**Pull request by uncenter**
cloudflareworkers

**Issue:** closes #8382.

**Similarweb rank:** not working for some reason, see original issue for metrics/popularity assessment.

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

The only thing I'm unsure of is the hex code. All of the icons on https://www.cloudflare.com/developer-platform/products/ use the `#FF6633` hex code, but the existing Cloudflare icons use
`#F38020` instead. I've also seen Cloudflare use the `#f4801f` color in some logos (e.g. https://www.cloudflare.com/logo/). Not sure how to best approach this, maybe I should just stick with what we have going and we can reconsider in a separate issue/PR?

### CODE REVIEW

**Comment by LitoMore** [comment]
The product Cloudflare Worker is part of Cloudflare. We could use the popularity of their main site as a reference.

**Comment by PeterShaggyNoble** [comment]
Although, this is now (barely) in scope on GH stars, its growth is [relatively flat](https://star-history.com/#cloudflare/workerd) compared to what it once was, so I'd say it would be out of scope. Is there anything else from our metrics we could use to illustrate its popularity? Even a Google Trends comparison with something similar that's already in our library or would be within our scope.

**Comment by uncenter** [comment]
Hmm. It's definitely one of Cloudflare's primary developer-focused products, and I'm confident it is quite popular. Would looking at npm downloads for their [`@cloudflare/workers-types` package](https://www.npmjs.com/package/@cloudflare/workers-types) count? It has roughly 480,000 weekly downloads.

File: _data/simple-icons.json [hunk 1]
```diff
@@ -2967,6 +2967,12 @@
             "source": "https://pages.cloudflare.com",
             "guidelines": "https://www.cloudflare.com/trademark/"
         },
+        {
+            "title": "Cloudflare Workers",
+            "hex": "FF6633",
```
Comment (PeterShaggyNoble) [inline _data/simple-icons.json:2972]:
```suggestion
            "hex": "F38020",
```

**Review by PeterShaggyNoble — APPROVED** [review]
Yep, that's definitely within our scope!

Let's run with our "standard" Cloudflare colour for now and we can have a separate conversation about whether it should be updated across the board on a per-icon basis. I'll leave you to open that issue as I'm about to disappear behind my bar for a weekend of touring rugby.

**Pull request merged**